### DISCUSSION TITLE
Does enabling iptables mean all ports are closed at boot time?

### DISCUSSION THREAD

**Discussion by pburris67** [gtxaspec/wz_mini_hacks#465] [General]
I set the IPTables directive to true, and I wondered if that means I need to issue iptables commands on the device to open up ports 22, 80, and the rtsp port (which I have forgotten because it's late in the day and my brain is fried)?  Or am I overthinking this?

Alternately, if I turn *OFF* iptables, will that open up the camera to anyone on my wifi network?

**Comment by endertable** [comment]
I believe enabling iptables will simply load the modules, you would still need to activate any filters or rules to allow or deny.  
Not enabling iptables, or not setting up filters with iptables enabled, will allow anyone on your LAN to access your open network resources that aren't protected through some other means.

**Comment by pburris67** [comment]
That’s exactly the response I sought. It means the camera behaves as I
would expect. Thank you for the rapid response. It helps. — Peter
